docs/upgrade/1.0.0_to_1.1.0.rst

```diff
@@ -60,109 +60,8 @@ Finally, run the following commands: ::
   ./securedrop-admin setup
   ./securedrop-admin tailsconfig
 
-.. _upgrade_to_tails_4:
+.. include:: ../upgrade_to_tails_4.rst
```

Comment:
The only reason we've kept this 1.0.0->1.1.0 upgrade guide around is that it includes the Tails 4 docs. Now that you've nicely split it off, we can do some pruning. I think you can safely prune all upgrade docs up to (not including) 1.3.0->1.4.0 as part of this PR.

Reply:
Removed in 60130e3d0